apps/dapp-console-api/src/constants/envVars.ts

```diff
@@ -88,6 +88,7 @@ const envVarSchema = z.object({
   FAUCET_ACCESS_ATTESTERS: z.string().array(),
   WORLDID_APP_ID: z.string(),
   WORLDID_APP_ACTION_NAME: z.string(),
+  API_QUEUE_REDIS_URL: z.string(),
```

Comment:
nit: can you add this env var to `example.env`?

Reply:
will fix!